syntax = "proto3";

package auth;

service AuthService {
    rpc Authenticate (AuthRequest) returns (AuthResponse);
}

message AuthRequest {
    string login = 1;
    string password = 2;
}

message AuthResponse {
    bool auth = 1;
}

